bundler.d/nulldb.rb

```diff
@@ -0,0 +1 @@
+gem 'activerecord-nulldb-adapter'
```

Comment:
This means we need to cp the packaging to rpm/1.24

Reply:
For DEB I'm planning to just remove that file in the build process.